### ISSUE TITLE
Exit Code 137 for newest release

### ISSUE THREAD

**Issue by moritz31** [aws/aws-for-fluent-bit#11]
We are triggering fargate tasks with a step function. For the newest fluentbit container this fails because the container always exits with exit code 137 when the actual container in the task finishs.
For the previous version 1.3.2 this is not a problem.

**Comment by PettitWesley** [comment]
@moritz31 I've reproduced this... it is very strange- 137 usually means Out of Memory AFAIK.

**Comment by PettitWesley** [comment]
I think I've got a fix for this.

One key thing to note- the old behavior (afaict) was that the container would always with code 0 even if Fluent Bit exited with a non-zero code. 

@moritz31 Is that what you've seen? My fix will restore that old behavior- the container will (afaict) always exit 0.

**Comment by PettitWesley** [comment]
I think I finally understand this.

137 means that the container did not gracefully handle the SIGTERM, and docker had to SIGKILL it: http://tldp.org/LDP/abs/html/exitcodes.html

Normally when Fluent Bit receives a SIGTERM, it prints a warning that it will quit soon- it has SIGTERM handling built in. I checked my logs and I noticed that with 2.0.0 this does not happen. This means that something about how I set up the entry point script prevents the signals from properly being passed to it. 

The `exec -c` I added in #14 fixes the signal passing somehow.
